NEWS

```diff
@@ -1,5 +1,17 @@
 systemd System and Service Manager
 
+CHANGES WITH 233 in spe
+
+        * The confirmation spawn prompt has been reworked to offer the
+          following choices:
+
+           (f)ail, don't execute the command and pretend it failed
+           (s)kip, don't execute the command and pretend it succeeded
+           (y)es, execute the command
+
+          The 'n' choice for the confirmation spawn prompt has been removed,
+          because its meaning was confusing.
```

Comment:
hmm, one more idea: maybe, when we read "n" we should print "Didn't understand 'n', did you mean 'f'?" and ask again?